.github/workflows/ci.yml

```diff
@@ -11,25 +11,17 @@ on:
 
 jobs:
     build:
-        runs-on: self-hosted-novum
-        container: docker.tuenti.io/service-inf/web-builder:pptr10.4-1.0.2
-
-        strategy:
-            matrix:
-                node-version: [16.x]
+        runs-on: ubuntu-latest
+        container: telefonicacdo/web-builder:chromium93.0.4577-1.2.3
 
         steps:
             - uses: actions/checkout@v2
 
-            - name: Use Node.js ${{ matrix.node-version }}
-              uses: actions/setup-node@v1
-              with:
-                  node-version: ${{ matrix.node-version }}
-
             - name: Install dependencies
               run: yarn install --immutable --immutable-cache
 
-            - run: yarn ci
+            - name: Run tests
```

Comment:
well, not only tests, but ok

Reply:
yes.. this is more like "Run everything" XD